lib/providers/services/firebase/push_notification_service.dart

```diff
@@ -51,3 +52,30 @@ class PushNotificationService {
     }
   }
 }
+
+Future<dynamic> backgroundMessageHandler(Map<String, dynamic> message, BuildContext context) async {
+  print("backgroundMessageHandler");
+  if (message.containsKey('data')) {
+    // Handle data message
+    final Map<dynamic, dynamic> data = message['data'];
+    print("onResume: data");
+    print("onResume: $data");
+
+    var notificationTypeId = data["notification_type_id"];
+    print("onResume notificationTypeId: $notificationTypeId");
+
+    if (notificationTypeId == guardianInviteReceived) {
+      //Navigate to Guardians Screen
+      NavigationService.of(context).navigateTo(Routes.guardianTabs);
+    }
+  }
+
+  if (message.containsKey('notification')) {
+    // Handle notification message
+    final dynamic notification = message['notification'];
+    print("onResume: notification");
+    print("onResume: $notification");
```

Comment:
I would recommend leaving print statements in the code only for errors - and in other situations only for weird stuff that's really hard to debug.. so I would be OK with a single print statement on a PN handler - that qualifies as hard to debug.